python/tests/wrapper/test_02_regularizers.py

```diff
@@ -0,0 +1,133 @@
+import os
+import itertools
+import tempfile
+import shutil
+import pytest
+
+import artm.wrapper
+import artm.wrapper.messages_pb2 as messages
+import artm.wrapper.constants as constants
+import helpers
+
+def test_func():
+    # Set some constants
+    dictionary_name = 'dictionary'
+    pwt = 'pwt'
+    nwt = 'nwt'
+    rwt = 'rwt'
+    docword = 'docword.kos.txt'
+    vocab = 'vocab.kos.txt'
+
+    smsp_phi_tau = -0.2
+    smsp_theta_tau = -0.1
+    decor_phi_tau = 1000000
+
+    num_topics = 10
+    num_inner_iterations = 10
+    num_outer_iterations = 8
+
+    perplexity_tol = 0.001
+    expected_perplexity_value_on_iteration = {
+        0: 6691.688,
+        1: 2442.690,
+        2: 2330.616,
+        3: 1826.080,
+        4: 1711.015,
+        5: 1614.403,
+        6: 1590.140,
+        7: 1564.405
+    }
+    sparsity_tol = 0.001
+    expected_phi_sparsity_value_on_iteration = {
+        0: 0.060,
+        1: 0.121,
+        2: 0.210,
+        3: 0.312,
+        4: 0.393,
+        5: 0.452,
+        6: 0.495,
+        7: 0.528
+    }
+    expected_theta_sparsity_value_on_iteration = {
+        0: 0.012,
+        1: 0.035,
+        2: 0.143,
+        3: 0.245,
+        4: 0.314,
+        5: 0.335,
+        6: 0.345,
+        7: 0.349
+    }
+
+    batches_folder = tempfile.mkdtemp()
+    try:
+        # Create the instance of low-level API and helper object
+        lib = artm.wrapper.LibArtm()
+        helper = helpers.TestHelper(lib)
+        
+        # Parse collection from disk
+        helper.parse_collection_uci(os.path.join(os.getcwd(), docword),
+                                    os.path.join(os.getcwd(), vocab),
+                                    batches_folder,
+                                    dictionary_name)
+
+        # Create master component and scores
```

Comment:
Consider adding a method that takes maps score config to its type (PerplexityScoreConfig -> ScoreConfig_Type_Perplexity, etc). I'm not sure what would be the most pythonic way --- either a function with switch, or a map from type to the constant. But it would be great to just specify config, and let helper.create_master_component automatically figure out the constant for the type.